lib/index.js

```diff
@@ -24,9 +24,10 @@ var Client    = require('client'),
 ///   console.log('Hi ' + currentUser.name);
 /// });
 /// ```
-ZAFClient.init = function(callback) {
-  var queryParams = Utils.queryParameters(),
-      hashParams = Utils.queryParameters(( document.location.hash || '' ).slice(1)),
+ZAFClient.init = function(callback, location) {
+  location = location || {};
```

Comment:
`location` here is shadowing `window.location`. Can we call it something else?

Reply:
so is `name` and `event`, but that doesn't seem to be a problem?

https://github.com/zendesk/zendesk_app_framework_sdk/blob/master/lib/client.js#L21
https://github.com/zendesk/zendesk_app_framework_sdk/blob/master/lib/client.js#L121

🤷‍♂️

Reply:
what would you suggest we change it to?

Reply:
Didn't know `name` was thing tbh. It's not critical, but would prefer to avoid it, especially when the real property is relevant in this context. Could change it to `loc` or `locationConfig`?